.github/workflows/ci.yml

```diff
@@ -392,7 +392,7 @@ jobs:
             awk 'BEGIN{FS="="} {print $3}' | sed s'/["}]//g')
           echo "DEFAULT_CONSTRAINTS_BRANCH=${DEFAULT_CONSTRAINTS_BRANCH}" >> $GITHUB_ENV
           DEBIAN_VERSION=$(grep "export DEBIAN_VERSION" scripts/ci/libraries/_initialization.sh | \
-            cut -d "=" -f 3 | sed s'/["}]//g')
+            awk 'BEGIN{FS="="} {print $3}' | sed s'/["}]//g')
```

Comment:
The output of this is the same, but let's be consistent across the board.

Reply:
Why not :).